xml/System.Numerics/INumberBase`1.xml

```diff
@@ -963,7 +963,7 @@ For <xref:System.Numerics.IFloatingPointIeee754%601> this method matches the IEE
         <param name="y">The value to compare with <paramref name="x" />.</param>
         <summary>Compares two values to compute which has the lesser magnitude and returning the other value if an input is <c>NaN</c>.</summary>
         <returns>
-          <paramref name="x" /> if it is less than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
+          <paramref name="x" /> if it has the lesser magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```

Comment:
```suggestion
          <paramref name="x" /> if it has a lesser magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```